### PR TITLE
Add env vars to build process.

### PR DESCRIPTION
**Pull request by jhocc**
Ran into deployment issues where assets were being built improperly with
an dev env.

[#131016183]

### CODE REVIEW

File: Dockerfile.production [hunk 1]
```diff
@@ -18,7 +18,8 @@ WORKDIR $APP_HOME
 ENV DISPLAY :1
 ADD . $APP_HOME
 ENV BUNDLE_PATH /ruby_gems
-
+ENV RAILS_ENV production
+ENV RACK_ENV production
```
Comment (sumancc) [inline Dockerfile.production]: What doe RACK_ENV do? Where is it used?

Comment (jhocc) [inline Dockerfile.production]: RACK_ENV is for other rack apps.  `RAILS_ENV` is all is needed for asset generation, but I put `RACK_ENV` incase any part of our stack needs it in the future.

Comment (sumancc) [inline Dockerfile.production]: Got it! Looks good then.

Comment (pemacc) [inline Dockerfile.production]: If you do not need it now, then you wont need it(YAGNI)

Comment (jhocc) [inline Dockerfile.production]: Fine... i'll remove it.

Comment (jhocc) [inline Dockerfile.production]: no biggie

**Pull request merged**